datadog_checks_dev/tests/tooling/commands/validate/test_package.py

```diff
@@ -0,0 +1,50 @@
+# (C) Datadog, Inc. 2022-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+import sys
+
+import pytest
+from click.testing import CliRunner
+
+from datadog_checks.dev import run_command
+
+
+def _build_pyproject_file(authors):
+    return f'''
+[project]
+name = "datadog-my-check"
+authors = {authors}
+[tool.hatch.version]
+path = "datadog_checks/my_check/__about__.py"
+'''
+
+
+@pytest.mark.parametrize(
+    'authors,expected_exit_code,expected_output',
+    [
+        ('[{ name = "Datadog", email = "packages@datadoghq.com" }]', 0, '1 valid'),
+        ('[{ name = "Datadog"}]', 0, '1 valid'),
```

Comment:
I can't say for sure, but the original ticket refers to builds failing if the email is invalid, and I've checked and an email is not required to successfully build a wheel, so that's why I was inclined to accept no email as valid.